File: ecm-core/src/main/java/br/com/samuel/app/resources/members/CountMembersResource.java
package br.com.samuel.app.resources.members;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import br.com.samuel.app.resources.interfaces.ICounterResource;
import br.com.samuel.app.usecases.members.CountMembers;
import br.com.samuel.app.usecases.models.CountElements;

@RestController
@RequestMapping("members")
public class CountMembersResource extends ICounterResource<CountMembers> {
    
    @GetMapping("/count")
    public ResponseEntity<CountElements> run() {
        return ResponseEntity.ok(counter().run());
    }
}

File: ecm-view/src/app/resources/members/count-members.resource.ts
import { Injectable } from "@angular/core";
import { Observable } from "rxjs";

import { ICounterResource } from "../interfaces/counter.resource";
import { CountElements } from "src/app/utils/models/count-elements.entity";

@Injectable({
    providedIn: 'root'
})
export class CountMembersResource extends ICounterResource {

    constructor() {
        super('members', 'count')
    }

    run(): Observable<CountElements> {
        return this.http.get<CountElements>(this.baseUrl());
    }
   
}